hathor/event/model/EventQueueOptions.py

```diff
@@ -0,0 +1,20 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from hathor.utils.pydantic import BaseModel
+
+
+class EventQueueOptions(BaseModel):
```

Comment:
Missing documentation for each parameter.

Reply:
Class removed in https://github.com/HathorNetwork/hathor-core/pull/608/commits/79f2773a773adcbef4f5d853344e51bb66cfb4d2

Reply:
Ok. Thanks!